Project/src/MakeCall/CallCard.js

```diff
@@ -318,22 +321,14 @@ export default class CallCard extends React.Component {
             this.setState(prevState => ({ remoteParticipants: [...prevState.remoteParticipants, participant] }));
         }
 
-        participant.on('displayNameChanged', () => {
-            console.log('displayNameChanged ', participant.displayName);
-        });
-
-        participant.on('stateChanged', () => {
-            console.log('Participant state changed', participant.identifier.communicationUserId, participant.state);
-        });
-
         const addToListOfAllRemoteParticipantStreams = (participantStreams) => {
             if (participantStreams) {
                 let participantStreamTuples = participantStreams.map(stream => { return { stream, participant, streamRendererComponentRef: React.createRef() }});
                 participantStreamTuples.forEach(participantStreamTuple => {
                     if (!this.state.allRemoteParticipantStreams.find((v) => { return v === participantStreamTuple })) {
-                        this.setState(prevState => ({
-                            allRemoteParticipantStreams: [...prevState.allRemoteParticipantStreams, participantStreamTuple]
-                        }));
+                        this.setState({
+                            allRemoteParticipantStreams: [...this.state.allRemoteParticipantStreams, participantStreamTuple]
```

Comment:
There was a good reason why we used prevstate here but it was a long time and i dont remember. Lets please make sure to test 1:1, escalation, group calls. In different paltforms

Reply:
Let me ask, why change it? Is it necessary? If not necessary, then better to just not change it

Reply:
It was having an array that was never cleaned and any new participant was pushed there without removing the participants that already left the call